docs/Splashkit/Applications/npm-installation-guide.md

```diff
@@ -0,0 +1,153 @@
+# Npm installation guide: Windows and Mac
+
+###### (scroll down for Mac instructions)
+
+### What does npm do?
+
+Npm is used to check markdown files (md) to match Prettier (plug-in) and needs Node.js to run properly.
+
+## Windows Guide
+
+---
+
+Open PowerShell and follow this guide: (installs WSL then WSL2)
+
+```
+ wsl --install
+```
+
+```
+dism.exe /online /enable-feature /featurename:Microsoft-Windows-Subsystem-Linux /all /norestart
+```
+
+```
+dism.exe /online /enable-feature /featurename:VirtualMachinePlatform /all /norestart
+```
+
+_Restart your computer_
+
+##### (this will take longer than usual as it is installing Linux for the first time)
+
+download the [WSL2 Kernel update](https://wslstorestorage.blob.core.windows.net/wslblob/wsl_update_x64.msi)
+
+_Run the installer_ (that you just downloaded)
+
+repond: _yes_
+
+return to PowerShell and set WSL2 as the default
+
+```powershell
+wsl --set-default-version 2
+```
+
+## Set up Node.js on WSL 2 [follow linked guide](https://docs.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-wsl)
+
+1. Open Ubuntu command line
+2. install cURL (used for downloading internet content in the command line)
+
+```powershell
+sudo apt-get install curl
+```
+
+1. Install nvm
+
+```powershell
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
+```
```

Comment:
**suggestion:** I am wondering if we could outsource these installation steps to the official [guide](https://docs.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-wsl#install-nvm-nodejs-and-npm). So in here, we just need to add an extra step like to following.

8. Follow the official [guide](https://docs.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-wsl#install-nvm-nodejs-and-npm) to install node.js and npm on WSL2.